solution/Makefile

```diff
@@ -177,7 +177,10 @@ test.pytest:
 	DJANGO_SETTINGS_MODULE="cmcs_regulations.settings.test_settings" docker-compose -f docker-compose.yml up -d
 	docker-compose exec regulations pytest -vvv
 	docker-compose -f docker-compose.yml up -d  ## resets the django_settings_module to the default
-
+test.pytest.cov:
+	DJANGO_SETTINGS_MODULE="cmcs_regulations.settings.test_settings" docker-compose -f docker-compose.yml up -d
```

Comment:
just a handy tool if you want tosee your code coverage percentage.  Example everyting in content_search is 92-100 percent covered